config.json

```diff
@@ -50,7 +50,11 @@
       {
         "advisory": "https://github.com/advisories/GHSA-mhxj-85r3-2x55",
         "issue": null
-      }
+      },
+      {
+        "advisory": "https://github.com/advisories/GHSA-36jr-mh4h-2g58",
+        "issue": null
+      },
```

Comment:
With the `,` here, this is not valid JSON

Reply:
fixed in https://github.com/brave/audit-config/pull/5/commits/7d095b2065b5315142d52aa746cba1f52b3ad1fc